src/Options.php

```diff
@@ -634,6 +656,7 @@ private function configureOptions(OptionsResolver $resolver)
             },
             'excluded_exceptions' => [],
             'excluded_app_paths' => [],
+            'send_default_pii' => false,
```

Comment:
Wouldn't be `send_pii` or `send_pii_by_default` better names? Maybe you can find a better one, what makes me wondering is that the current one seems to tell users that the SDK will send some default PII which to me means nothing, or at least I don't understand what it means

Reply:
May I suggest to change this in the unified API then? I won't block this PR, just suggesting a change :wink: